src/main/kotlin/uk/gov/gdx/datashare/services/ScheduledJobService.kt

```diff
@@ -16,17 +17,25 @@ class ScheduledJobService(
   private val acquirerEventRepository: AcquirerEventRepository,
   private val groApiService: GroApiService,
   private val supplierEventRepository: SupplierEventRepository,
-  meterRegistry: MeterRegistry,
+  private val meterRegistry: MeterRegistry,
+  private val outboundEventQueueService: OutboundEventQueueService,
 ) {
-  private val gauge =
+  private val unconsumedEventsGauge =
     meterRegistry.gauge("UnconsumedEvents", AtomicInteger(acquirerEventRepository.countByDeletedAtIsNull()))
```

Comment:
Not for now, but perhaps we ought to split this out like we have the queues? So per subscription

Reply:
WIll do

Reply:
https://govukverify.atlassian.net/browse/GPC-397?atlOrigin=eyJpIjoiZTJlMmM4OTE5OGY3NGQzZjkzYjg2MDdhODM0ZjhkYjAiLCJwIjoiaiJ9